python/rpdk/python/codegen.py

```diff
@@ -251,6 +252,7 @@ def _docker_build(cls, external_path):
                 auto_remove=True,
                 volumes=volumes,
                 stream=True,
+                user="{uid}:{gid}".format(uid=os.geteuid(), gid=os.getgid()),
```

Comment:
this should probably be a format string like on line 241

Reply:
I guess there’s a couple ways to address this. The {} style is preferred in Python3. I can update the rest of the file to use Python3 format strings or I can change this spot to use the Python2 format to be consistent. It’s up to you.

Reply:
the `{}`'s are fine, but you can achieve the same thing with `f"{os.geteuid()}:{os.getgid()}"` instead of having to type `.format( /* ... */ )`.

Reply:
got it. just submitted an updated PR